phpstan-baseline.neon

```diff
@@ -2085,6 +2085,11 @@ parameters:
 			count: 2
 			path: src/Core/Content/Product/SalesChannel/Detail/AvailableCombinationResult.php
 
+		-
+			message: "#^Cannot call method fetchAll\\(\\) on Doctrine\\\\DBAL\\\\Driver\\\\Statement\\|int\\.$#"
```

Comment:
please do not add new things here. 

this should also not be necessary
https://github.com/shopware/platform/blob/master/phpstan.neon.dist#L38

Reply:
Must've been a rebase issue

Reply:
Thanks for the hint. I removed it :)